mobile.css

```diff
@@ -0,0 +1,412 @@
+  padding-bottom: 12px;
+}
+
+.about-me p {
+  color: #42526e;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  margin-bottom: 24px;
+}
+
+.about-me h2 {
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: block;
+  margin: 70px auto 24px auto;
+}
+
+.vital {
+  padding: 60px 0;
+}
+
+.strength {
+  width: 90%;
+  height: 352px;
+  background: #ebf0ee;
+  border: 1px solid #c1c7d0;
+  border-radius: 8px;
+  margin: 0 auto;
+  margin-bottom: 24px;
+}
+
+.strength-img {
+  width: 80px;
+  height: 80px;
+  background: #fff;
+  border-radius: 50%;
+  margin: 0 auto;
+  margin-top: 40px;
+  margin-bottom: 64px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.strength .strength-p {
+  display: flex;
+  justify-content: center;
+  color: #344563;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+}
+
+.strength li:hover {
+  background: #a3cabc;
+  border: 1px solid #fff;
+}
+
+.strength .ul-1 {
+  margin: 20px 0 8px 0;
+}
+
+.strength .ul-2 {
+  margin-bottom: 24px;
+}
+
+.footer {
+  min-width: 100vw;
+  margin: 0 auto;
+  margin-bottom: 10px;
+  background-image: url('./images/illustration-contact-form.svg');
+  background-position: right 68%;
+  background-repeat: no-repeat;
+}
+
+.footer .text {
+  width: 92%;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  text-align: center;
+  margin: 60px auto;
+}
+
+input,
+textarea {
+  font-family: 'Inter', sans-serif;
+  width: 90%;
+  display: block;
+  color: #6b778c;
+  margin: 0 auto;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  padding: 15px 16px;
+  border-radius: 4px;
+  margin-bottom: 24px;
+  border: 1px solid #d0d9d4;
+  transition: all 0.3s;
+}
+
+input::-webkit-input-placeholder {
+  color: #6b778c;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+form button {
+  color: #fff;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border-radius: 4px;
+  background: #36b37f;
+  padding: 12px;
+  border-style: none;
+  display: block;
+  width: 130px;
+  margin: 0 auto;
+}
+
+form button:hover {
+  cursor: pointer;
+  background: #36b37f;
+}
+
+form button:active {
+  cursor: pointer;
+  background: #008552;
+}
+
+.email {
+  display: flex;
+  justify-content: center;
+  margin: 10px auto 336px auto;
+}
+
+.bar {
+  width: 150px;
+  border-bottom: 10px solid rgb(36, 35, 35);
+  margin: 0 auto;
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.space {
+  height: 10px;
+}
+
+/* styles for mobile menu project */
+.click-mobile-icon {
+  transform: translateY(0);
+}
+
+.options {
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  z-index: 99;
+  background: #fff;
+  width: 100vw;
+  height: 100vh;
+  padding-left: 24px;
+  position: fixed;
+  overflow-x: hidden;
+  overflow-y: hidden;
+}
```

Comment:
- [OPTIONAL ] I can see that you use `position: fixed`, However it has not applied yet, and I think it's about specification case. Specificity is the means by which browsers decide which CSS property values are the most relevant to an element and, therefore, will be applied,  That will help preventing your mobile navigation menu to scroll.
- For more information you can check [How is specificity calculated?](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#how_is_specificity_calculated)